### DISCUSSION TITLE
im confuse how .env work

### DISCUSSION THREAD

**Discussion by ibrahimtrg18** [vercel/next.js#22172] [Help] [answered]
![image](https://user-images.githubusercontent.com/41787706/107880806-c845a000-6f13-11eb-9254-e045b42f8bd7.png)
![image](https://user-images.githubusercontent.com/41787706/107880833-e3181480-6f13-11eb-8f45-a0221af13e2a.png)

i already set the NODE_ENV to production
why load env from .env.development.local? instead of load from .env.production.local

where did i do wrong here?

thanks.

**Comment by michrome** [comment, marked as answer]
Hi @ibrahimtrg18 

Check out this [priority table](https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use). If `.env.development.local` exists, it will take priority over everything else.
